qcodes/instrument_drivers/Spectrum/M4i.py

```diff
@@ -587,21 +589,17 @@ def __init__(self, name, cardid='spcm0', **kwargs):
     # checks if requirements for the compensation get and set functions are met
     def _get_compensation(self, i):
         # if HF enabled
-        if(getattr(self, 'input_path_{}'.format(i))() == 1):
-            self._param32bit(
-                getattr(pyspcm, 'SPC_ACDC_OFFS_COMPENSATION{}'.format(i)))
-        else:
-            logging.warning(
-                "M4i: HF path not set, ignoring ACDC offset compensation get\n")
+        logging.info("M4i: HF path not set, ignoring ACDC offset compensation get\n")
+        return self._param32bit(
```

Comment:
@peendebak I don't understand :) so you removed the "if" statement here? If so, why?) and since the "if" statement is removed, why did you leave the "logging.info" which is only applicable to the case where HF is disabled? Also `# if HF enabled` comment is now obsolete? Could you explain?

Reply:
You are right, I will put the `if` back for the `logging.info`

Reply:
ok, but now the "logging.info" message is misleading because "ACDC offset compensation parameter" is NOT ignored, because the "return" statement is still executed. Could you fix the "logging.info" message?

Reply:
@astafan8 The parameter is returned in the method, but it is ignored by the M4i card. See page 65 of https://spectrum-instrumentation.com/sites/default/files/download/m4i44_manual_english.pdf.

I changed the info message a bit to make this clear.